newsService/src/main/java/com/kovan/app/service/NewsService.java

```diff
@@ -63,21 +71,52 @@ public NewsDto getTopHeadlines() {
 
         } while (pageSize <= newsDto.getTotalResults());
```

Comment:
Do Refactor this do ... while loop try to incorporate java 8 feature.

Reply:
Refactored the do while loop.